go/libraries/doltcore/sqle/tables.go

```diff
@@ -297,6 +328,14 @@ func (t *WritableDoltTable) Updater(ctx *sql.Context) sql.RowUpdater {
 	return te
 }
 
+// GetAutoIncrementValue gets the last AUTO_INCREMENT value
+func (t *WritableDoltTable) GetAutoIncrementValue(ctx *sql.Context) (interface{}, error) {
```

Comment:
See my comment above. How is go-mysql-server expected to respond to a table that declares that it supports auto increment values via the interface method, but doesn't declare any auto_inc columns? Needs to be part of the contract. You could make it explicit by adding a `HasAutoIncrementColumn` method to that interface, or you could sort it out with good error handling. But what you have now is kind of confusing.

(also I don't think you should need to implement this method on WritableDoltTable, since it embeds DoltTable)